### ISSUE TITLE
Build all files in directory

### ISSUE THREAD

**Issue by nottrobin** [sass/sassc#176]
Unless I've missed something, it seems that currently `sassc` only supports compiling one file at a time. Is there any plan to support building every file in a directory?

**Comment by xzyfer** [comment]
There are currently no plans for active feature development on SassC. We're open to pull requests.

**Comment by nottrobin** [comment]
@xzyfer ah. Has the project been abandoned/deprecated, or is it just that the project lacks leaders?

Am I wrong in thinking this is the most direct way to run [LibSass](http://sass-lang.com/libsass), which I thought was intended to be the future standard version of Sass?

**Comment by xzyfer** [comment]
> Has the project been abandoned/deprecated

No. We've purposely opted to keep it simple and dependency free. 

> is it just that the project lacks leaders

Our primary focus is LibSass feature parity. We consider this feature complete for 95% of users. We're open to pull requests but we're not actively working on this.

> Am I wrong in thinking this is the fastest way to run

You are not mistaken. Which is another reason we prefer to keep things simple. 
Alternative you can use node-sass which is more actively maintained.

>I thought was intended to be the future standard version of Sass?

There are no offical plans for this. Using LibSass is a choice :)

**Comment by nottrobin** [comment]
@xzyfer thanks for so thoroughly answering each of my questions :+1:

**Comment by ZaDarkSide** [comment]
+1

**Comment by funkyfuture** [comment]
it would would be helpful if there was an option that would derive the output path from the input path by simply replacing the file extension, that would make it easy to use with `find` like:

    find . -name "*.scss" -exec sassc --that-neded-option {} \;

**Comment by saper** [comment]
Currently `sassc file.scss` without a second argument prints CSS to stdout. Should this be changed? Good question.

**Comment by funkyfuture** [comment]
i don't think so, as it would break current usages.
